liblangutil/ParserBase.cpp

```diff
@@ -122,7 +122,7 @@ void ParserBase::expectTokenOrConsumeUntil(Token _value, char const *_lhs, bool
 		while (token != _value && token != Token::EOS)
 			token = m_scanner->next();
 		std::string const expectToken = ParserBase::tokenName(_value);
-		std::string const mess = fmt::sprintf("In <%s>, %s is expected; got %s instead.", _lhs, expectToken.c_str(), ParserBase::tokenName(tok).c_str());
+		std::string const mess = "In <" + _lhs + ">, " + expectToken + " is expected; got " + ParserBase::tokenName(tok)" instead";
```

Comment:
@chriseth this doesn't work as is because + doesn't seem  to work between `const char [5]` and `const char *` . The actual message I get is: 

```
solidity/liblangutil/ParserBase.cpp: In member function ‘void langutil::ParserBase::expectTokenOrConsumeUntil(langutil::Token, const char*, bool)’:
solidity/liblangutil/ParserBase.cpp:128:35: error: invalid operands of types ‘const char [5]’ and ‘const char*’ to binary ‘operator+’
   std::string const mess = "In <" + _lhs + ">, " + expectToken + " is expected; got " + ParserBase::tokenName(tok)" instead";
```
I will come back an look at this at another time.

Reply:
Addressed in d94f5ff .